level_2/section_4.md

```diff
@@ -1,84 +1,30 @@
-# Section 4: More Advanced Survival Techniques
-
-## Enchanting
+# Section 4: Enchanting
 
 Enchanting is a mechanic that augments armor, tools, weapons and books with one or more of a variety of enchantments that improve an item's existing abilities or imbue them with additional abilities and uses.
```

Comment:
Is it okay to use "fancy" words like `augments` and `imbue`? I'm wondering this because the README (at the root of the repo) says level 2 is for 6-12 year olds.